Kernel/Arch/aarch64/init.cpp

```diff
@@ -92,6 +94,35 @@ ALWAYS_INLINE static Processor& bootstrap_processor()
 
 Atomic<Graphics::Console*> g_boot_console;
 
+static void init_stage2(void*);
+void init_stage2(void*)
+{
+    Process::register_new(Process::current());
+
+    // This thread is created to show that kernel scheduling is working!
+    LockRefPtr<Thread> more_work_thread;
+    (void)Process::create_kernel_process(more_work_thread, MUST(KString::try_create("More Work Thread"sv)), [] {
+        dmesgln("Enter loop (more work):");
+        for (int i = 0;; i++) {
+            if (i % 20 == 0)
```

Comment:
Should we have one of the two terminate after a while, to test that scenario? (Assuming that works :))

Reply:
That's indeed a very good suggestion! Will change that!